database/java/2017/5662.yaml

```diff
@@ -0,0 +1,40 @@
+---
+cve: 2017-5662
+title: CVE in org.apache.xmlgraphics:batik-svg-dom
+description: >
+    In Apache Batik before 1.9, files lying on the filesystem of the server which uses batik can be revealed to arbitrary users who send maliciously formed SVG files. The file types that can be shown depend on the user context in which the exploitable application is running. If the user is root a full compromise of the server - including confidential or sensitive files - would be possible. XXE can also be used to attack the availability of the server via denial of service as the references within a xml document can trivially trigger an amplification attack.
+cvss_v2: 7.9
+references:
+    - http://www.oracle.com/technetwork/security-advisory/cpuoct2017-3236626.html
+    - http://www.securityfocus.com/bid/97948
+    - http://www.securitytracker.com/id/1038334
+    - https://access.redhat.com/errata/RHSA-2017:2546
+    - https://access.redhat.com/errata/RHSA-2017:2547
+    - https://access.redhat.com/errata/RHSA-2018:0319
+    - https://xmlgraphics.apache.org/security.html
+
+affected:
+    - groupId: org.apache.xmlgraphics
+      artifactId: batik-svg-dom
+      version:
+        - "<= 1.0,1.8"
```

Comment:
The whitespace shouldn't be there and the version should be <=greater,lower, like in "all versions equal or lower than 'greater' are affected, all the way down to 'lower' version."

Reply:
https://github.com/victims/victims-cve-db/

Reply:
updated @msrb Thanks